controllers/infra/drivers/aws/get_instance.go

```diff
@@ -126,6 +126,9 @@ func (d Driver) getInstancesByLabel(key string, value string, infra *v1.Infra) (
 
 	for _, r := range result.Reservations {
 		for _, i := range r.Instances {
+			if i.State.Name == types.InstanceStateNameShuttingDown || i.State.Name == types.InstanceStateNameTerminated || i.State.Name == types.InstanceStateNameStopping || i.State.Name == types.InstanceStateNameStopped {
```

Comment:
Instances in the stopped or stopping state cannot be ignored. Shutting down and terminated will auto clean by aws

Reply:
Terminated aws will cause an error if we want to get its PrivateIpAddress when the aws terminates but is not cleaned.